fv_tts:
  ros__parameters:
    default_voice: ""
    output_topic: "audio/tts/frame"
    playback_topic: "audio/output/frame"
    use_playback_topic: true
    cache_dir: "~/.fluent_voice_cache"
    default_volume_db: -10.0
